packages/react-router-website/modules/components/ExampleRouter.js

```diff
@@ -1,7 +1,40 @@
-import React from 'react'
+import React, { PropTypes } from 'react'
 
-const ExampleRouter = ({ children }) => (
-  children ? React.Children.only(children) : null
-)
+class ExampleRouter extends React.Component {
+
+  static contextTypes = {
+    router: PropTypes.object.isRequired
+  }
+
+  static childContextTypes = {
+    router: PropTypes.object.isRequired
+  }
+
+  getChildContext() {
+    return {
+      router: this.router
+    }
+  }
+
+  componentWillMount() {
+    this.router = {
+      ...this.context.router,
+      match: null
+    }
+
+    this.unlisten = this.router.listen(() => {
+      this.router.location = this.context.router.location
```

Comment:
I had to modify `<ExampleRouter>` in order for it to work with relative routes. The `<FakeBrowser>` renders everything inside of a pathless `<Route>`, so without this relative `<Route>`s and `<Link>`s would resolve using that `<Route>`'s `match`. This just sets the `<ExampleRouter>`'s `match` to `null`. It also has to listen for location changes because any of its children that listen for location changes would be re-rendering via the `listener` prior to `<ExampleRouter>`'s `getChildContext` being called on navigation.